src/rules/typescript-requiring-type-checking/extensions.ts

```diff
@@ -0,0 +1,28 @@
+const suggestions = require('../eslint/suggestions').rules;
+
+export = {
+  parser: '@typescript-eslint/parser',
+  plugins: ['@typescript-eslint'],
+  rules: {
+    '@typescript-eslint/dot-notation': suggestions['dot-notation'],
+    '@typescript-eslint/no-implied-eval': suggestions['no-implied-eval'],
+    '@typescript-eslint/no-throw-literal': suggestions['no-throw-literal'],
+    '@typescript-eslint/require-await': suggestions['require-await'],
```

Comment:
This will in fact disable `@typescript-eslint/require-await` as it is disabled in `suggestions`.